fusor-ember-cli/app/routes/review/installation.js

```diff
@@ -37,6 +37,9 @@ export default Ember.Route.extend({
             return (sub.get('quantity_to_add') && sub.get('source') == 'added');
         });
         controller.set('reviewSubscriptions', reviewSubscriptions);
+        controller.set('hasSubscriptionsToAttach', reviewSubscriptions.get('length') > 0);
+        controller.set('hasSessionPortal', Ember.isPresent(this.modelFor('subscriptions')));
+        controller.set('hasSubscriptionPools', Ember.isPresent(this.controllerFor('subscriptions/select-subscriptions').get('subscriptionPools')));
```

Comment:
if a user refreshes the `review/installation` page, then the model hooks on route `subscriptions` and `subscriptions/select-subscriptions` are `undefined`. Since the `attachSubscriptions()` action needs this info, we need to alert the user to go back to these routes and re-enter (or refresh) the info. And then when the user comes back to the `review/installation` page, those model hooks will have run, so user can now click "Deploy"